test/csv/csv-filter/test_csv_filter.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: false
+
+require_relative "../helper"
+
+class TestFilter < Test::Unit::TestCase
+
+  def setup
+    @input = [
+      %w[aaa bbb ccc].join(","),
+      %w[ddd eee fff].join(","),
+      "" # Force trailing newline.
+    ].join("\n")
+  end
+
+  # Return filepath of file containing CSV data.
+  def csv_filepath(input, dirpath, option)
+    filename = "#{option}.csv"
+    filepath = File.join(dirpath, filename)
+    File.write(filepath, input)
+    filepath
+  end
+
+  # Return stdout and stderr from CLI execution.
+  def run_csv_filter(filepath, *cli_option_names)
+    top_dir = File.join(__dir__, "..", "..", "..")
+    command_line = [
+      Gem.ruby,
+      "-I",
+      File.join(top_dir, "lib"), 
+      File.join(top_dir, "bin", "csv-filter"),
+      *cli_option_names,
+      filepath,
+    ].join(" ")
+    Tempfile.create("stdout", mode: File::RDWR) do |stdout|
+      Tempfile.create("stderr", mode: File::RDWR) do |stderr|
+        system(command_line, {1 => stdout, 2 => stderr})
+        stdout.rewind
+        stderr.rewind
+        [stdout.read, stderr.read]
+      end
+    end
+  end
+
+  # Return results for CLI-only option (or invalid option).
+  def results_for_cli_option(option_name)
```

Comment:
Can we use better name? This executes `csv-filter` not a simple getter. So we should use a verb not a noun (`results`) for method name. (I think that we can unify this and `run_csv_filter` and use `run_csv_filter` for it...)